src/couch/src/couch_file.erl

```diff
@@ -264,6 +265,19 @@ rename_file(Original) ->
         Else -> Else
     end.
 
+rename_dir(RootDelDir, Original, DbName) ->
```

Comment:
This function does not seem to be handling databases that may have multiple path elements (ie, a DbName with a '/' in it) like you do for cloudant-labs/hastings#3.